integration-cli/docker_cli_rm_test.go

```diff
@@ -19,6 +20,9 @@ func (s *DockerSuite) TestRmContainerWithRemovedVolume(c *check.C) {
 	}
 	defer os.RemoveAll(tempDir)
 
+	out, _, _ := dockerCmdWithStdoutStderr(c, "ps", "-a")
+	fmt.Printf("out: %s", out)
```

Comment:
Wouldn't a c.Log get printed out only when there is an assertion failure? Is the goal to see what's ps'd in all cases or only the failure cases? 

What is RunConf?